content/ko/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md

```diff
@@ -0,0 +1,121 @@
+---
+title: cgroup 드라이버 설정
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+이 페이지에서는 kubelet cgroup 드라이버를 컨테이너
+런타임 cgroup 드라이버와 일치하도록 구성하는 방법을 설명한다.
+
+## {{% heading "prerequisites" %}}
+
+쿠버네티스 [컨테이너 런타임 요구사항](/ko/docs/setup/production-environment/container-runtimes)을
+숙지하고 있어야 한다.
+
+<!-- steps -->
+
+## 컨테이너 런타임 cgroup 드라이버 구성
+
+[컨테이너 런타임](/ko//docs/setup/production-environment/container-runtimes)
+페이지에서는 kubeadm이 `systemd` 서비스로 kubelet을 관리하기 때문에
+`cgroupfs` 대신에 systemd 드라이버를 사용하는 것이 좋다고 설명한다.
+
+이 페이지에서는 `systemd` 드라이버를 기본으로 사용하여 다양한 컨테이너 런타임을 설정하는 방법에 대한
+자세한 내용도 제공한다.
+
+## kubelet cgroup 드라이버 구성
+
+kubeadm은 `kubeadm init` 도중에 `KubeletConfiguration` 구조를 전달할 수 있다.
+이 `KubeletConfiguration`에는 kubelet의 cgroup 드라이버를 제어하는 `cgroupDriver`
+필드가 포함될 수 있다.
+
+{{< note >}}
+
+v1.22에서는 사용자가 `KubeletConfiguration`에서 `cgroupDriver` 필드를 설정하지 않은 경우
+`kubeadm`는 기본으로 `systemd`를 사용한다.
+{{< /note >}}
+
+필드를 명시적으로 구성하는 최소한의 예시이다.
+
+```yaml
+# kubeadm-config.yaml
+kind: ClusterConfiguration
+apiVersion: kubeadm.k8s.io/v1beta3
+kubernetesVersion: v1.21.0
+---
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
+cgroupDriver: systemd
+```
+
+위와 같은 구성 파일을 kubeadm 명령에 전달할 수 있다.
+
+```shell
+kubeadm init --config kubeadm-config.yaml
+```
+
+{{< note >}}
+Kubeadm은 클러스터의 모든 노드에서 동일한 `KubeletConfiguration`을 사용한다.
+`KubeletConfiguration`은 `kube-system` 네임스페이스에 속한 [컨피그맵](/ko/docs/concepts/configuration/configmap)
+오브젝트에 저장된다.
+
+하위 명령인 `init`, `join`, `upgrade`를 실행하면 kubeadm이
+`/var/lib/kubelet/config.yaml`에 있는 `KubeletConfiguration`을 파일로
+작성하여 로컬 노드 kubelet에 전달한다.
+{{< /note >}}
+
+## `cgroupfs` 드라이버 사용
+
+`cgroupfs`를 사용하고 `kubeadm upgrade`가 기존 설정의
+`KubeletConfiguration` cgroup 드라이버를 수정하는 것을 방지하려면, 해당
+값을 명시적으로 표시해야 한다. 이는 향후 버전의 kubeadm에서 기본적으로
+`systemd` 드라이버를 적용하지 않으려는 경우에 해당된다.
```

Comment:
[원문]
> To use cgroupfs and to prevent kubeadm upgrade from modifying the KubeletConfiguration cgroup driver on existing setups, you must be explicit about its value. This applies to a case where you do not wish future versions of kubeadm to apply the systemd driver by default.

```suggestion
`cgroupfs`를 사용하고 `kubeadm upgrade`가 기존 설정의
`KubeletConfiguration` cgroup 드라이버를 수정하는 것을 방지하려면, 해당
값을 명시적으로 표시해야 한다. 이는 향후 버전의 kubeadm이 기본적으로
`systemd` 드라이버를 적용하는 것을 원하지 않는 경우에 해당된다.
```

Reply:
좀 더 자연스러운 표현을 알아갑니다. 감사합니다!